src/helix/hooks.cljc

```diff
@@ -42,8 +53,13 @@
            updater (react/useCallback (fn updater
                                         ([x] (u x))
                                         ([f & xs]
```

Comment:
I don't think you need the 1-arity case anymore, since it'll be handled by the multi-arity case?

I'm actually a bit confused that this works at all since the 2nd case can take 1 or more arguments.

Reply:
hmm the the 2nd case gets called if there are 2 or more arguments only, since there's a 1-arity case already, but you're right i could get rid of it. 🤔 

that would mean that we could use multimethods which take no arguments too.... i like it